### DISCUSSION TITLE
Data link - data point time range

### DISCUSSION THREAD

**Discussion by darthkurak** [grafana/grafana#60890] [Dashboards, Panels & Visualizations]
Hi. I wanted to implement a simple use case with a Time Series chart.

In TimeSeries, each point represents an aggregation of data in a specific time range. When I click on any data point, I want to open another dashboard narrowed to the time range of clicked data point. It would work like a kind of drill-down or "zoom-in".
For instance, when my aggregation (interval) is set to one day, after clicking a point on the chart, I would like to open another dashboard with the time range set to that exact day I clicked. It will show all "raw data" in a table, which previously was counted in aggregation. 
Unfortunately, I didn't find any possible way to do that. 
We have `__value.time` variable. If we treat it as "to", there is no way to calculate "from" from it and pass to the dashboard url.  
syntax ` ${__value.time} - ${__interval} ` doesn't work. 
Would be great to have some way to achieve this :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82897. Feel free to continue the discussion around this there. Thank you!
